javascript/pull-view.js

```diff
@@ -33,6 +33,11 @@
         this.$el.addClass("thumbsup");
       }
 
+      this.wip = (this.model.get('title').indexOf('WIP') >= 0);
```

Comment:
Would it be possible to extend this to include other strings, like "DO NOT MERGE"?

The seal slack bot has a bunch of them: https://github.com/binaryberry/seal/blob/master/config/alphagov.yml

It's team specific there, but there's probably a common subset that would work quite well.